### PR TITLE
Add back git requirements as comments & re-run script

### PR DESCRIPTION
**Pull request by AlanCoding**
##### SUMMARY
Making alternative to https://github.com/ansible/awx/pull/15316

After diving into the issue further, I believe the approach in 15316 is untenable. Yes, it fixes things a _little_ bit, but the output still doesn't make sense.

By going this route, and turning those into a comment, we have a fully complete file.

~TODO: we should add a short comment explaining why we do this.~

##### ISSUE TYPE
 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME
 - API

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -77,6 +78,7 @@ botocore==1.34.47
     #   s3transfer
 cachetools==5.3.2
     # via google-auth
+# git+https://github.com/ansible/system-certifi.git@devel  # git requirements installed separately
     # via
     #   -r /awx_devel/requirements/requirements_git.txt
     #   kubernetes
```
Comment (AlanCoding) [inline requirements/requirements.txt:90]: Now this is coherent output.

File: requirements/requirements.txt [hunk 2]
```diff
@@ -512,6 +533,7 @@ typing-extensions==4.9.0
 urllib3==1.26.18
     # via
     #   botocore
+    #   django-ansible-base
```
Comment (AlanCoding) [inline requirements/requirements.txt:536]: This is real, @john-westcott-iv added `urllib3` as a dependency in https://github.com/ansible/django-ansible-base/pull/633

File: requirements/requirements.txt [hunk 3]
```diff
@@ -478,6 +495,10 @@ slack-sdk==3.27.0
     # via -r /awx_devel/requirements/requirements.in
 smmap==5.0.1
     # via gitdb
+social-auth-app-django==5.4.2
+    # via django-ansible-base
+social-auth-core==4.5.4
+    # via social-auth-app-django
```
Comment (AlanCoding) [inline requirements/requirements.txt:501]: @john-westcott-iv also from https://github.com/ansible/django-ansible-base/pull/633, via resource_registry, so updater script is correct.

However, I'm a little concerned about this.

**Comment by AlanCoding** [comment]
Failures...

```
 license for requirement defusedxml is missing
 license for requirement python3-openid is missing
 license for requirement social-auth-app-django is missing
 license for requirement social-auth-core is missing
```

**Review by simaishi — APPROVED** [review]

**Review by TheRealHaoLiu — APPROVED** [review]

**Review by TheRealHaoLiu — APPROVED** [review]

**Pull request merged**